CMakeLists.txt

```diff
@@ -20,8 +20,7 @@ project(rdycore)
 set(PETSC_DIR $ENV{PETSC_DIR})
 set(PETSC_ARCH $ENV{PETSC_ARCH})
 find_package(PETSc COMPONENTS C)
-set(ENV{PKG_CONFIG_PATH} ${PETSC_DIR}/${PETSC_ARCH}/lib/pkgconfig)
-list(APPEND CMAKE_PREFIX_PATH $ENV{PKG_CONFIG_PATH})
+set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:${PETSC_DIR}/${PETSC_ARCH}/lib/pkgconfig")
 
```

Comment:
@jeff-cohere I hope you're okay with this. The other prevents using libCEED that isn't configured by PETSc.